spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestComputeTableStatsAction.java

```diff
@@ -79,6 +79,20 @@ public class TestComputeTableStatsAction extends CatalogTestBase {
           required(4, "nestedStructCol", NESTED_STRUCT_TYPE),
           required(5, "stringCol", Types.StringType.get()));
 
+  @TestTemplate
+  public void testLoadingTableDirectly() {
```

Comment:
This test would previously fail.

Reply:
Suggestion: Should we move this test to `org.apache.iceberg.spark.TestSparkTableUtil`

Reply:
I feel it belongs here as it is important to check the action can be invoked without loading tables via the Spark catalog (as that one will set the catalog name correctly).